CMakeLists.txt

```diff
@@ -232,30 +232,18 @@ if(openPMD_HAVE_HDF5)
     target_link_libraries(openPMD.io PUBLIC ${HDF5_LIBRARIES})
     target_include_directories(openPMD.io SYSTEM PUBLIC ${HDF5_INCLUDE_DIRS})
     target_compile_definitions(openPMD.io PUBLIC ${HDF5_DEFINITIONS})
-    target_compile_definitions(openPMD.io PUBLIC "-DLIBOPENPMD_WITH_HDF5=ON")
-    if(openPMD_HAVE_MPI)
-        # TODO: remove and just rely on "WITH_MPI"
-        target_compile_definitions(openPMD.io PUBLIC "-DLIBOPENPMD_WITH_PARALLEL_HDF5=ON")
-    endif()
+    target_compile_definitions(openPMD.io PUBLIC "-DopenPMD_HAVE_HDF5=ON")
```

Comment:
@C0nsultant as pre-processor macros, `1` (ON) and `0` (OFF) are usually better than strings since you can build `#if`s on those

Reply:
Currently the preprocessor logic relies on a variable being defined.
I agree that a boolean check is better since it conveys the right intention (present or not)